src/modules/logicModule/LM_PC_FundingPot_v1.sol

```diff
@@ -472,18 +478,18 @@ contract LM_PC_FundingPot_v1 is
     }
 
     /// @dev    Validate uint start input.
-    /// @param  _start uint to validate.
-    /// @param  _cliff uint to validate.
-    /// @param  _end uint to validate.
+    /// @param  start_ uint to validate.
+    /// @param  cliff_ uint to validate.
+    /// @param  end_ uint to validate.
     /// @return True if uint is valid.
-    function _validTimes(uint _start, uint _cliff, uint _end)
+    function _validTimes(uint start_, uint cliff_, uint end_)
```

Comment:
Expose this in the exposed file and test, do this for all internal functions. Fuzz tests please :)